tests/models/translationproject.py

```diff
@@ -0,0 +1,81 @@
+import os
+import shutil
+
+import pytest
+
+from django.db import IntegrityError
+
+from pootle_language.models import Language
+from pootle_project.models import Project
+from pootle_translationproject.models import TranslationProject
+
+
+@pytest.mark.django_db
+def test_tp_create_fail(tutorial, english):
+
+    # trying to create a TP with no Project raises a RelatedObjectDoesNotExist
+    # which can be caught with Project.DoesNotExist
```

Comment:
Nit: I don't particularly like texts starting with lowercase and without ending period. Every time I see one of these I have doubts about if part of the comment has been removed at the beginning or at the end, or in other words if comment is incomplete.